nlp_primitives/tests/test_universal_sentence_encoder.py

```diff
@@ -20,9 +21,9 @@ def test_regular(universal_sentence_encoder):
         "Mitochondria is the powerhouse of the cell",
     ])
     a = pd.DataFrame(universal_sentence_encoder(sentences))
-    a = a.mean().round(7).astype('str')
-    b = pd.Series(['-0.0007475', '0.0032088', '0.0018552', '0.0008256', '0.0028342'])
-    assert a.equals(b)
+    a = a.mean().round(7).to_numpy()
+    b = pd.Series([-0.0007475, 0.0032088, 0.0018552, 0.0008256, 0.0028342])
```

Comment:
good catch here, weird that I (at least I think I was the culprit here) had cast it to a string to compare.